main.cpp

```diff
@@ -214,7 +218,14 @@ void Bass::playOpenStrings(int numStrings)
 
     for (int i = 0; i < numStrings; i++)
     {
-        std::cout << circleOfFourths[(index + i) % 12] << std::endl;
+        std::cout << circleOfFourths[(index + i) % 12];
+
+        if (i == numStrings - 1)
+        ;
+        else
+        {
+            std::cout << ", ";
+        }
```

Comment:
This is confusing to look at.   What are you trying to have happen here? 
are you going for this?
```
if( i < numStrings -1 ) 
    std::cout << ", "; 
```

Reply:
You're absolutely right. I don't know what I was thinking.